tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/message/payments/PaidMediaToTelegramPaidMedia.kt

```diff
@@ -0,0 +1,27 @@
+package dev.inmo.tgbotapi.types.message.payments
+
+import dev.inmo.tgbotapi.types.media.TelegramPaidMedia
+import dev.inmo.tgbotapi.types.media.TelegramPaidMediaPhoto
+import dev.inmo.tgbotapi.types.media.TelegramPaidMediaVideo
+
+fun PaidMedia.toTelegramPaidMediaOrNull(): TelegramPaidMedia? = when (this) {
```

Comment:
It is better to replace this function __after__ `toTelegramPaidMediaVideo` and `toTelegramMediaPhoto` and use them in function instead of duplicating their code